plugins/woocommerce/includes/admin/meta-boxes/views/html-product-attribute-inner.php

```diff
@@ -37,10 +37,17 @@
 
 				if ( 'select' === $attribute_taxonomy->attribute_type ) {
 					$attribute_orderby = ! empty( $attribute_taxonomy->attribute_orderby ) ? $attribute_taxonomy->attribute_orderby : 'name';
+				  /**
+				   * Filter the length (number of terms) rendered in the list.
+	               *
+				   * @since 8.5.0
```

Comment:
Please update to `8.6.0`: 

```suggestion
				   * @since 8.6.0
```